### ISSUE TITLE
How to ‘hack’ and win the May Mayhem blog contest

### ISSUE THREAD

**Issue by AlexVanderbist** [laravel/blog-contest-may-mayhem#57]
https://alexvanderbist.com/2018/how-to-hack-and-win-the-may-mayhem-blog-contest

**Comment by ghost** [comment]
@AlexVanderbist  Congratulations for the most creative article! 💯

**Comment by freekmurze** [comment]
😈

**Comment by kfirba** [comment]
Haha, what a great article! Good one!

Oh, of-course :+1:

**Comment by introwit** [comment]
@alexgiuvara described it really well, damn creative 💯

**Comment by m1guelpf** [comment]
We have a winner!

**Comment by MarcoSpoor** [comment]
Definitely creative 😎

**Comment by laraning** [comment]
Excellent article, very creative and well written. Also, it's inside the "contest rules" since it shows how to integrate the GitHub with an Laravel app, so it's Laravel blog scope! Congrats!

**Comment by llanderr** [comment]
Great writing and creative approach!

**Comment by castroalves** [comment]
The 🏆 Oscar goes to... @AlexVanderbist! Well done!

**Comment by jobcerto** [comment]
LOL, With this tutorial i can make a bunch of tasks in my apps with Dusk. It's awesome!

**Comment by JosielSantos** [comment]
Great article! A very creative tutorial that shows many features and facilities of Laravel
and educate people that exposes your keys in public repositories.

**Comment by hamzaali00001** [comment]
what if someone really used this method and mess up votes on everyone's issue. Everyone would look fake. Also, this post has the highest number of votes :stuck_out_tongue_winking_eye: . I think someone is going to mess things up.

**Comment by AlexVanderbist** [comment]
Thank you all for the kind words and feedback! This is my second technical blogpost I've ever written so it's nice to see all time and effort pay off :D 

@hamzaali00001 I was sort of expecting this to happen before I even wrote the blogpost. However, I'm not too worried this will mess voting up more than people secretly asking their friends to vote 😅

**Comment by davorminchorov** [comment]
Clever use of contest rules :D

**Comment by brendt** [comment]
@AlexVanderbist 

> so it's nice to see all time and effort pay off

I see what you did there 💰

**Comment by RemiCollin** [comment]
Brilliant post; you managed to expose the fact that github reactions are not a reliable metric for this kind of contest, and teach something useful at the same time. You get my thumb up, which is not a fake one, FWIW :)

**Comment by ghost** [comment]
i voted

**Comment by connor11528** [comment]
I've never used Dusk for testing but after reading this definitely want to get started! Seems like lots of opportunities for web scraping use cases also. 👏 @AlexVanderbist

**Comment by Insolita** [comment]
@connor11528 The usage dusk on server more difficult, than on local machine. For web scrapping the best decision is puppeteer now https://github.com/GoogleChrome/puppeteer It more lightweight. You may find php -wrappers for it usage, as one of example is https://github.com/spatie/browsershot

**Comment by AlexVanderbist** [comment]
@Insolita Thanks for the great suggestions! spatie/browsershot is pretty great 🤷‍♂️

**Comment by mnapoli** [comment]
This is a really great article. However how does it work regarding the "no lobby" rule? Because of its content the article mentions the contest, so it attracts people here to vote.

In any case all votes here seem legit, 99% of the votes are from accounts that existed *before* the contest started :p (see https://ihy0lrdsek.execute-api.us-east-1.amazonaws.com/dev for an analysis, and [here](https://gist.github.com/mnapoli/e344b243b378cbe36a7ee71e18582c47) if you want to see the source code)

**Comment by AlexVanderbist** [comment]
@mnapoli thanks! I wasn't sure how it would work with the "no lobby" rule (didn't expect this article to blow up like it did). However, it's my first "released" blogpost and the url to the blog hasn't been shared at all outside of this issue. This is what traffic over the last month looks like ¯\_(ツ)_/¯ 
![image](https://user-images.githubusercontent.com/6287961/40825063-bc3c9556-6576-11e8-9f07-a916b5a1e4ab.png)

**Comment by geoffreyvanwyk** [comment]
Some folks used the wrong reaction emoticons, unless they did it intentionally.

**Comment by usamamuneerchaudhary** [comment]
haha, loved the idea 💯Congrats to the winner already 🥇🥇

**Comment by taylorotwell** [comment]
It looks like this is the winner 👍 

@AlexVanderbist can you email me at taylor at laravel with your Paypal address? :)

**Comment by AlexVanderbist** [comment]
@taylorotwell yes, thank you so much! 🎉 I'll contact you once I get home tonight :)

**Comment by JunaidQadirB** [comment]
Congratulations @AlexVanderbist !

**Comment by mmikhan** [comment]
Very creative and clever. You deserve this @AlexVanderbist 👍

**Comment by calebporzio** [comment]
Haha, well played sir.. nice post!

**Comment by duncanmcclean** [comment]
Well Done @alexvanderbist 👍 Great Post

**Comment by lorisleiva** [comment]
@AlexVanderbist Absolutely love your blog's design 👨‍🎨 Well done 👍

**Comment by Norris1z** [comment]
Great post @AlexVanderbist you really deserve this. 👍

**Comment by martinbean** [comment]
👏

**Comment by kamerk22** [comment]
Congratulation @AlexVanderbist 👏
